src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,156 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int INITIAL_LIST_SIZE = 10;
+    private static final int FIRST_INDEX = 1;
+    private int size;
+    private T[] elementData;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        Node(Node<T> prev, T item, Node<T> next) {
+            this.prev = prev;
+            this.item = item;
+            this.next = next;
+        }
+    }
+
+    MyLinkedList() {
+        first = new Node<T>(null, null, null);
+        last = new Node<T>(null, null,null);
+    }
```

Comment:
Fields first and last should be fields of class without any declaration. You don't need this constructor.

Reply:
Deleted